xbmc/pvr/PVRManager.cpp

```diff
@@ -1121,6 +1121,25 @@ bool CPVRManager::PlayMedia(const CFileItem& item)
   return g_application.PlayFile(pvrItem, false) == PLAYBACK_OK;
 }
 
+void CPVRManager::UpdateCurrentChannel(void)
+{
+  CSingleLock lock(m_critSection);
+
+  if (m_currentFile)
+  {
+    CPVRChannelPtr channel(m_currentFile->GetPVRChannelInfoTag());
+    CPVRChannelPtr playingChannel(GetCurrentChannel());
+
+    if (channel && playingChannel && (*channel != *playingChannel))
```

Comment:
would probably make sense to just add a method `bool CPVRManager::IsPlayingChannel(const CPVRChannelPtr& channel)` because we have the same check in multiple places